srcpkgs/logisim-evolution/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'logisim-evolution'
+pkgname=logisim-evolution
+version=3.8.0
+revision=3
+hostmakedepends="openjdk17 gradle sed tar"
+depends="openjdk17-jre"
+short_desc="Digital logic design tool and simulator"
+maintainer="voidbert <humbertogilgomes@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/logisim-evolution/logisim-evolution"
+changelog="https://github.com/logisim-evolution/logisim-evolution/raw/master/CHANGES.md"
+distfiles="https://github.com/logisim-evolution/logisim-evolution/archive/refs/tags/v${version}.tar.gz"
+checksum=7a49558a490a0df886b3ea488f12e4ea887a2ec8c278872dca39b2bebfaed3ee
+
+do_patch() {
+	sed -i -e "s:Icon=.*:Icon=${DESTDIR}/usr/share/applications:g" \
+	       -e "s:Exec=.*:Exec=${DESTDIR}/usr/bin/logisim-evolution:g" \
+	       support/Flatpak/com.github.reds.LogisimEvolution.desktop
+}
+
+do_build() {
+	export JAVA_HOME=/usr/lib/jvm/openjdk17
+	gradle --no-daemon shadowJar
+}
+
+do_install() {
+	vmkdir usr/share/logisim-evolution
+	vmkdir usr/share/applications
+	vmkdir usr/bin
+
+	vcopy build/libs/logisim-evolution-${version}-all.jar usr/share/logisim-evolution
+
+	sed -e "s/@VERSION@/${version}/g" ${FILESDIR}/logisim-evolution > \
+		${DESTDIR}/usr/bin/logisim-evolution
+	chmod 0755 ${DESTDIR}/usr/bin/logisim-evolution
+
+	vcopy artwork/logisim-evolution-icon.svg usr/share/logisim-evolution
```

Comment:
use vinstall, and this should go in /usr/share/icons/hicolor/scalable/apps/ with a name that matches the Icon= line in the desktop file